config.js

```diff
@@ -35,6 +36,10 @@ const DEFAULT_DB_CONNECTION_POOL = { // sequelize defaults
   idle: 10000,
 };
 
+// Enable Rooms Routes
+const swagger = featureToggles.isFeatureEnabled('enableRooms') ?
+  './api/v1/rooms-swagger.yaml' : './api/v1/swagger.yaml';
```

Comment:
Depends on whether we will ever want to give a refocus admin the choice to turn on/off the rooms feature